packages/server/src/cli.ts

```diff
@@ -12,6 +12,7 @@ import { SharedProcess, SharedProcessState } from "./vscode/sharedProcess";
 import { setup as setupNativeModules } from "./modules";
 import { fillFs } from "./fill";
 import { isCli, serveStatic, buildDir } from "./constants";
+import opn = require("opn");
```

Comment:
We have to use `import = require()` style because opn's type definition is written in `export =` style.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/opn/index.d.ts#L12-L15
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require